### PR TITLE
Create paper projections for paper similarity graph

### PR DESCRIPTION
**Pull request by georgepar**
The 3 steps of #54 

Changelog:

- Add initial script `scripts/create_papers_csv.py` to create a first version of `papers.csv`. This was needed to proceed with this. @GuyAglionby will take over the next version of `papers.csv`

- Modify `scripts/reduce.py` to include UMAP projection method

- Modify `templates/poster.html` and `main.py` to add code for viewing similar papers

- Add docs for the process

Also see the PR from sitedata repo:
https://github.com/acl-org/acl-2020-virtual-conference-sitedata/pull/3

### CODE REVIEW

**Comment by georgepar** [comment]
@hao-fang I've also created a branch in the sitedata private repo with the updated files. 
Not sure what to do so we can see this on CI-deployed site.

Currently one needs to grab the updated [papers.csv](https://github.com/acl-org/acl-2020-virtual-conference-sitedata/blob/add_acl2020_accepted_papers_tsv/papers.csv) and [papers_projection.json](https://github.com/acl-org/acl-2020-virtual-conference-sitedata/blob/add_acl2020_accepted_papers_tsv/papers_projection.json) and paste them into `sitedata_acl2020` and run the site locally.

**Comment by georgepar** [comment]
Also I see in the CI `make format-check` fails.
I've run `make format` and `make format-check` locally and it passes. Any ideas?

File: templates/poster.html [hunk 1]
```diff
@@ -123,6 +123,33 @@ <h5 style="color: red;">
...
+        <div class="pp-card-header" class="text-muted">
+          <a href="poster_{{openreview.content.iclr_id}}.html" class="text-muted">
+            <h5 class="card-title" align="center">{{openreview.content.title}}</h5>
+          </a>
+          <h6 class="card-subtitle text-muted" align="center">
+             {% for a in openreview.content.authors %}
+             {{a}},
+             {% endfor %}
+          </h6>
+          <center><img class="cards_img" src="https://iclr.github.io/iclr-images/{{openreview.content.iclr_id}}.png" width="80%"/></center>
```
Comment (hao-fang) [inline templates/poster.html:146]: Can we make `https://iclr.github.io/iclr-images` a config variable and use `{{ config.xxxx_url }}` here?
Then we can edit the `sitedata_acl2020/config.yml` file to add this field.

**Comment by hao-fang** [comment]
> @hao-fang I've also created a branch in the sitedata private repo with the updated files.
> Not sure what to do so we can see this on some CI-deployed site.
> 
> Currently one needs to grab the updated [papers.csv](https://github.com/acl-org/acl-2020-virtual-conference-sitedata/blob/add_acl2020_accepted_papers_tsv/papers.csv) and [papers_projection.json](https://github.com/acl-org/acl-2020-virtual-conference-sitedata/blob/add_acl2020_accepted_papers_tsv/papers_projection.json) and paste them into `sitedata_acl2020` and run the site locally.

1. Can you make a PR in the sitedata private repo and I'll approve?
2. After we merge that PR, you can go to this repo locally, run
    ```bash
    cd sitedata_acl2020
    git pull
    cd ../
    git add sitedata_acl2020
    git commit -am "bump sitedata_acl2020"
    git push
    ```
This way it should point to the latest private sitedata.

**Comment by hao-fang** [comment]
> Also I see in the CI `make format-check` fails.
> I've run `make format` and `make format-check` locally and it passes. Any ideas?

It says
```
ERROR: /home/runner/work/acl-2020-virtual-conference/acl-2020-virtual-conference/scripts/create_papers_csv.py Imports are incorrectly sorted.
```

It seems that your local python venv is not synced with the CI venv.
Did you follow the instruction [here](https://github.com/acl-org/acl-2020-virtual-conference/blob/master/CONTRIBUTING.md) to create your local virtualenv?

File: templates/poster.html (hunk 1 above)
Comment (georgepar) [reply templates/poster.html:146]: Updated with latest commit

**Comment by georgepar** [comment]
> Did you follow the instruction [here](https://github.com/acl-org/acl-2020-virtual-conference/blob/master/CONTRIBUTING.md) to create your local virtualenv?

I will try it. Thanks :)

**Comment by georgepar** [comment]
Ok I did some refactoring to simplify code, since papers.csv was updated in sitedata repo.

Now the CI fails in make freeze because paper_recs.json is missing so I think this PR should be merged first.

https://github.com/acl-org/acl-2020-virtual-conference-sitedata/pull/3/files

**Comment by georgepar** [comment]
@hao-fang after bumping sitedata checks pass, so this can be merged after review

**Pull request merged**